src/config.c

```diff
@@ -2971,6 +2971,7 @@ standardConfig static_configs[] = {
     /* Unsigned int configs */
     createUIntConfig("maxclients", NULL, MODIFIABLE_CONFIG, 1, UINT_MAX, server.maxclients, 10000, INTEGER_CONFIG, NULL, updateMaxclients),
     createUIntConfig("unixsocketperm", NULL, IMMUTABLE_CONFIG, 0, 0777, server.unixsocketperm, 0, OCTAL_CONFIG, NULL, NULL),
+    createUIntConfig("socket-id", NULL, MODIFIABLE_CONFIG, 0, UINT_MAX, server.soid, 0, INTEGER_CONFIG, NULL, NULL),
```

Comment:
Should be `IMMUTABLE_CONFIG`?
Otherwise, we can't make the config work unless we execute `config set bind *` after.